src/library_idbfs.js

```diff
@@ -195,28 +210,190 @@ mergeInto(LibraryManager.library, {
+				}
+			};
+			req.onerror = function(e) {
+				callback(this.error);
+				e.preventDefault();
+			};
+		}
+		var req = store.get(path);
+		req.onsuccess = function(event) 
+		{
+			if (typeof event.target.result === 'undefined' )
+			{
+				var contents=[];
+				loadRemoteMulti(store, path, contents,0,0,callback);
+			}
+			else
+			{
+				callback(null, event.target.result); 
+			}
+		};
+		req.onerror = function(e) {
+			callback(this.error);
+			e.preventDefault();
+		};
     },
-    storeRemoteEntry: function(store, path, entry, callback) {
-      var req = store.put(entry, path);
-      req.onsuccess = function() { callback(null); };
-      req.onerror = function(e) {
-        callback(this.error);
-        e.preventDefault();
-      };
+    storeRemoteEntry: function(store, path, entry, callback) 
+	{
+		function storeRemoteEntryMulti(store, path, entry,contents, index,total, callback) 
+		{
+			if (index>0)
+			{
+				entry.contents=contents.slice((index-1)*IDBFS.DB_MAXRECORDSIZE,index*IDBFS.DB_MAXRECORDSIZE);
+			}
+			else
+			{
+				entry.contents=contents;
+			}
+			var req = store.put(entry, path+IDBFS.DB_PARTSSUFFIX+index+total);
+			req.onsuccess = function() 
+			{
+				if (index>0 && index<total)
+				{
+					storeRemoteEntryMulti(store, path, entry,contents,index+1,total, callback);
+				}
+				else
+				{
+					callback(null);
+				}
+			};
+			req.onerror = function(e) 
+			{
+				callback(this.error);
+				e.preventDefault();
+			};
+		}
+		
+		var clength=(entry && entry.contents)?entry.contents.length:0;
+		var nbParts=Math.ceil(clength/IDBFS.DB_MAXRECORDSIZE);
+		var req;
+		if (nbParts>1 && nbParts<10)
+		{
+			var contents=entry.contents;
+			entry.contents=new entry.contents.constructor(1);
+			entry.contents[0]=nbParts;
+			req = store.put(entry, path+IDBFS.DB_PARTSSUFFIX+'00');
+			req.onsuccess = function() 
+			{
+				storeRemoteEntryMulti(store, path, entry,contents,1,nbParts, callback);
+			};
+		}
+		else
+		{
+			req = store.put(entry, path);
+			req.onsuccess = function() 
+			{
+				callback(null);
+			};
+		}
+		req.onerror = function(e) 
+		{
+			callback(this.error);
+			e.preventDefault();
+		};
     },
     removeRemoteEntry: function(store, path, callback) {
-      var req = store.delete(path);
-      req.onsuccess = function() { callback(null); };
-      req.onerror = function(e) {
+		function remoteDelete(store,path,index,total,callback)
+		{
+			var addstr=(index>=0?IDBFS.DB_PARTSSUFFIX+index+total:'');
+			  var req = store.delete(path+addstr);
+			  req.onsuccess = function() 
+			  { 
+				if (total>0 && index>0)
+				{
+					remoteDelete(store,path,index-1,total,callback);
+				}
+				else if (total>0 && index===0 )
+				{
+					remoteDelete(store,path,0,0,callback);
+				}
+				else
+				{
+					callback(null); 
+				}
+			  };
+			  req.onerror = function(e) {
+				callback(this.error);
+				e.preventDefault();
+			  };
+		}
+/*		function remoteExists(store,path,cbk)
```

Comment:
please remove commented-out code

Reply:
Hi, Thanks for formatting the code.
Sorry, I am new on GitHub and not familliar with the flow. Are you expecting something from me to validate?
Regards.

Reply:
No problem. Please fix the formatting, and make a commit with that, and then push that commit to the branch of this PR. Then the commit will show up in the PR and I can review it here.